File: Java/1304. Find N Unique Integers Sum up to Zero.java
class Solution {
    public int[] sumZero(int n) {
        int[] arr = new int[n];
        boolean isOdd =false;
        if (n%2!=0){
            isOdd= true;
            n--;
        }
        for (int i =0;i<n/2;i++){
            arr[i] = i+1;
            arr[n-i-1]= -(i+1);
        }
        if (isOdd){
            arr[n]=0;
        }
        return arr;
    }
}

File: Python/1304. Find N Unique Integers Sum up to Zero.py
class Solution:
    def sumZero(self, n: int) -> List[int]:
        arr = []
        isOdd = False
        if (n%2!=0):
            isOdd = True
            n-=1
        for i in range(n//2):
            arr.append(i+1)
            arr.append(-(i+1))
        if isOdd:
            arr.append(0)
        return arr